Description: Revert upstream's disabling of -Bsymbolic-functions on non-x86
 The claimed binutils upstream fix is already in utopic:
 https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git;h=97323ad11305610185a0265392cabcd37510f50e
Author: Timo Jyrinki <timo.jyrinki@canonical.com>

--- b/config.tests/unix/bsymbolic_functions.test
+++ a/config.tests/unix/bsymbolic_functions.test
@@ -6,10 +6,6 @@
 
 
 cat >>bsymbolic_functions.c << EOF
-#if !(defined(__i386) || defined(__i386__) || defined(__x86_64) || defined(__x86_64__) || defined(__amd64))
-#error "Symbolic function binding on this architecture may be broken, disabling it (see QTBUG-36129)."
-#endif
-
 int main() { return 0; }
 EOF
 
